Sequential.lua

```diff
@@ -23,6 +23,16 @@ function Sequential:insert(module, index)
    self.gradInput = self.modules[1].gradInput
 end
 
+function Sequential:remove(index)
+   index = index or #self.modules
+   if index > #self.modules or index < 1 then
+      error"index out of range"
+   end
+   table.remove(self.modules, index)
+   self.output = self.modules[#self.modules].output
```

Comment:
Nevermind, it should work as is. `updateOutput` and `updateGradInput` take care of what I said earlier. My bad !